pom.xml

```diff
@@ -61,7 +61,7 @@
     </dependencyManagement>
     <dependencies>
         <dependency>
-            <groupId>${project.groupId}</groupId>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
```

Comment:
Can we mark a comment or `TODO` here? Otherwise I feel like this will get reverted by some well-meaning individual and some maintainer (including myself) will not realize and let it through.

Reply:
I don't understand what kind of comment you want to see. Referring to group IDs by their literal form rather than an interpolated string is the norm, not the exception. Moreover, I don't see what action there is to take, so I'm not sure what kind of TODO to write.

Reply:
> Referring to group IDs by their literal form rather than an interpolated string is the norm, not the exception

That was my confusion. I thought it was the other way around, so I thought someone in the future might accidentally clean up the pom and give you trouble down the road. If that's the case,  then no action needed.